wallet/wallet.c

```diff
@@ -4677,6 +4677,7 @@ bool wallet_offer_create(struct wallet *w,
 	db_query_prepared(stmt);
 
 	if (db_step(stmt)) {
+		db_col_ignore(stmt, "1");
```

Comment:
`SELECT 1` is a strange way of saying `SELECT COUNT(*)` :-) The latter would return a single row, whose only column we're interested in. Anyway, this is a reorg, not a cleanup, so I'll roll this into my weekly cleanup PR ^^

Reply:
Hm, I wonder if there's a difference between `COUNT(1)` and `COUNT(*)`. Using stars in queries always makes me a bit uneasy, haha.